src/Operation/Find.php

```diff
@@ -353,6 +338,11 @@ private function createCommandDocument(): array
             return $cmd;
         }
 
+        // Read concern can change the query plan
+        if (isset($this->options['readConcern'])) {
```

Comment:
I think this conditional can be removed.

`createQueryOptions()` actually includes the `readConcern` option, so it will already be added by the final `$cmd + $options` union in the return statement. This is because the `readConcern` option is passed to the Query constructor instead of `Server::executeQuery()` (as it might be for `executeReadCommand()` in other operations).

Hypothetically, if that wasn't the case, the position of this conditional would still be problematic due to the early return above if `createQueryOptions()` returns an empty array. The edge case would be constructing Find with only a `readConcern` option.

While writing this, I realized the docs regarding read concern inheritance for queries is not very clear. Opened [PHPC-2249](https://jira.mongodb.org/browse/PHPC-2249) to improve that.